### PR TITLE
Better Rails example instructions

### PR DESCRIPTION
**Pull request by Cawllec**
## Goal

Updates the examples and configuration required to run Sidekiq & Que in the `rails-51` example application.

### CODE REVIEW

File: example/rails-51/config/database.yml [hunk 1]
```diff
@@ -5,6 +5,11 @@
 #   gem 'sqlite3'
 #
 default: &default
+  adapter: sqlite3
+  pool: 5
+  timeout: 5000
```
Comment (tobyhs) [inline example/rails-51/config/database.yml]: (GitHub won't let me comment further below) The database entry for each section using the sqlite3 adapter should ideally be a file in the db directory with an extension of sqlite3 (as opposed to just "default" in the Rails root).

File: example/rails-51/README.md [hunk 2]
```diff
@@ -64,10 +70,17 @@ Once PostgreSQL is set up as detailed using [the PostgreSQL documentation](https
...
+
+```shell
+export RAILS_ENV=que
+```
+
 Ensure that the initial Que setup is complete by running:
 
 ```shell
-bundle exec bin/rake que:install
+bundle exec bin/rails generate que:install
```
Comment (tobyhs) [inline example/rails-51/README.md]: In this example app, running db:migrate on the migration generated by running this command fails for me:
```
bugsnag-ruby/example/rails-51$ RAILS_ENV=que be rake db:migrate
rake aborted!
StandardError: An error has occurred, this and all later migrations canceled:

Directly inheriting from ActiveRecord::Migration is not supported. Please specify the Rails release the migration was written for:

  class AddQue < ActiveRecord::Migration[4.2]
```

We might need que 0.14.x for it to work (unless I'm doing something wrong).

Comment (tomlongridge) [reply example/rails-51/README.md]: I get the same...

File: example/rails-51/config/database.yml (hunk 1 above)
Comment (tomlongridge) [reply example/rails-51/config/database.yml]: We don't make use of sqlite database in our example as far as I can see, so don't think we need to update this.

Comment (tomlongridge) [reply example/rails-51/config/database.yml]: We don't make use of these sqlite databases in our examples as far as I can see - I don't see any benefit in updating this.

File: lib/bugsnag/integrations/que.rb [hunk 3]
```diff
@@ -1,3 +1,5 @@
+require 'que'
```
Comment (tobyhs) [inline lib/bugsnag/integrations/que.rb]: It looks like the addition of this line causes spec/integrations/que_spec.rb to fail. If you intend to keep this line, you'll likely need to add que to the test group of the root Gemfile and possibly remove the manual Que mock in que_spec.rb.

**Pull request merged**